src/core/kmod-setup.c

```diff
@@ -85,10 +85,6 @@ int kmod_setup(void) {
                 /* This should never be a module */
                 { "unix",      "/proc/net/unix",            true,   true,    NULL      },
 
-#if HAVE_LIBIPTC
-                /* netfilter is needed by networkd, nspawn among others, and cannot be autoloaded */
-                { "ip_tables", "/proc/net/ip_tables_names", false,  false,   NULL      },
-#endif
```

Comment:
distros that keep this enabled shouldn't mind about this auto-loading, no?

Reply:
Ok, fair enough.  I've tossed the commit that dropped the ip_tables autoload from kmod-setup. I've retained the snipped that enables the --port option unconditionally and merged that part into the preceeding change.